src/Controller/ListController.php

```diff
@@ -96,4 +97,19 @@ public function getMembersAction(Request $request, SubscriberList $list): View
 
         return View::create()->setData($list->getSubscribers());
     }
+
+    /**
+     * Gets the total number of subscribers of a list.
+     *
+     * @param Request $request
+     * @param SubscriberList $list
+     *
+     * @return View
+     */
+    public function getCountsubscribersAction(Request $request, SubscriberList $list): View
```

Comment:
This should be camelCase: `getCountsubscribersAction` -> `getCountSubscribersAction`

Reply:
That's what I tried before, It doesn't work camelCase.

Reply:
What exactly was the result?

Reply:
@oliverklee "Not Found" if you try /lists/1/countsubscribers.
It happens because the routes are generated automatically based on the Case.
There are some options here: 
1- Keep getCountsubscribersAction as function name to access the number of subscribers with 
/lists/1/countsubscribers
2- Change it to getCountSubscribersAction to access it with /lists/1/count/subscribers
3- Implement manual route generation. https://symfony.com/doc/current/bundles/FOSRestBundle/7-manual-route-definition.html
Thoughts?

Reply:
@xh3n1 Can we have route like `/lists/1/subscribers/count/`? This would be the correct one as far as URL semantics go, I think.

Reply:
It works for me. @samtuke what do you think?

Reply:
Sounds good